doxigen and opencl dependencies?

Great tool to have a nice clean full install of ImageMagick . During the install there where 2 errors that maybe stopped me from having a successful install.
1. (indicated in the log) regarding doxigen that was seamingly solved with (an apology i didn't write down the error)

>sudo apt install doxygen

2. it was looking for :

> ./MagickWand/studio.h:133:12: fatal error: CL/cl.h: No such file or directory
and it finally went away after doing the following

> sudo apt install opencl-headers

now I got the happy message:
![image](https://user-images.githubusercontent.com/41617013/175400202-54048709-0bfb-48fd-8802-d46a5d22a8eb.png)
the leftover errors seem to be of no impact to the software usage.
![image](https://user-images.githubusercontent.com/41617013/175400364-43bcfcec-17df-42a9-8e31-9c425da689bc.png)
eddit: now I see those are not "errors"... lol

No biggy, it might be a faulty messy set up of mine, just letting you guys know.

sidenote: I came back looking for an updated version of your script from 2019 to install ImageMagick!

--- a/imei.sh
+++ b/imei.sh
@@ -526,16 +526,16 @@ install_aom() {
         fi
 
         # see https://github.com/SoftCreatR/imei/issues/9
-        CMAKE_FLAGS="-DBUILD_SHARED_LIBS=1 -DENABLE_DOCS=0 -DENABLE_TESTS=0 -DENABLE_CCACHE=1"
+        CMAKE_FLAGS=(-DBUILD_SHARED_LIBS=1 -DENABLE_DOCS=0 -DENABLE_TESTS=0 -DENABLE_CCACHE=1)
 
         if [[ "${OS_DISTRO,,}" == *"raspbian"* ]]; then
-          CMAKE_FLAGS+=' -DCMAKE_C_FLAGS="-mfloat-abi=hard -march=armv7-a -marm -mfpu=neon"'
+          CMAKE_FLAGS+=(-DCMAKE_C_FLAGS="-mfloat-abi=hard -march=armv7-a -marm -mfpu=neon")
         fi
 
         tar -xf "aom-$AOM_VER.tar.gz" &&
           mkdir "$WORK_DIR/build_aom" &&
           cd "$WORK_DIR/build_aom" &&
-          cmake "../aom-$AOM_VER/" $CMAKE_FLAGS &&
+          cmake "../aom-$AOM_VER/" "${CMAKE_FLAGS[@]}" &&
           make
 
           if [ -n "$CHECKINSTALL" ]; then

--- a/imei.sh
+++ b/imei.sh
@@ -8,7 +8,7 @@
 # Author         : Sascha Greuel <hello@1-2.dev>             #
 # Date           : 2022-08-31 09:01                          #
 # License        : ISC                                       #
-# Version        : 6.6.3                                     #
+# Version        : 6.6.4                                     #
 #                                                            #
 # Usage          : bash ./imei.sh                            #
 ##############################################################
@@ -535,7 +535,7 @@ install_aom() {
         tar -xf "aom-$AOM_VER.tar.gz" &&
           mkdir "$WORK_DIR/build_aom" &&
           cd "$WORK_DIR/build_aom" &&
-          cmake "../aom-$AOM_VER/" "$CMAKE_FLAGS" &&
+          cmake "../aom-$AOM_VER/" $CMAKE_FLAGS &&
           make
 
           if [ -n "$CHECKINSTALL" ]; then

